### PR TITLE
Increase maximum ad notifications per day for tier 1 and 2 regions

### PR DESCRIPTION
**Pull request by tmancey**

### CODE REVIEW

File: seed/seed.json [hunk 1]
```diff
@@ -323,6 +323,34 @@
                 "channel": ["NIGHTLY"],
                 "platform": ["WINDOWS", "MAC", "LINUX"]
             }
+        },
+        {
+            "name": "AdServingIncreaseMaximumAdNotificationsPerDayTo60ForTier1And2Regions",
+            "experiments": [
+                {
+                    "name": "AdServing",
```
Comment (iefremov) [inline seed/seed.json:331]: I'd suggest we use "Enabled" or something similar as the experiment name

this is how it looks on brave://version
```
DoNotShowCustomAdNotificationsOnBetaAndReleaseChannelsStudy:AdNotifications
EphemeralStorageStudy:Enabled
PermissionLifetimeStudy:Enabled
```

File: seed/seed.json [hunk 2]
```diff
@@ -323,6 +323,34 @@
                 "channel": ["NIGHTLY"],
                 "platform": ["WINDOWS", "MAC", "LINUX"]
             }
+        },
+        {
+            "name": "AdServingIncreaseMaximumAdNotificationsPerDayTo60ForTier1And2Regions",
```
Comment (iefremov) [inline seed/seed.json:328]: Does it make sense to specify `60` in the study name? We could modify the parameter in the future. 
maybe just `MaxAdNotificationsPerDayStudy`?

[the remark this answers is not in the document]
Comment (moritzhaller) [reply seed/seed.json]: confirmed that this is not needed as discussed with @tmancey

File: seed/seed.json (hunk 1 above)
Comment (moritzhaller) [reply seed/seed.json]: fixed as requested by @tmancey

File: seed/seed.json (hunk 2 above)
Comment (moritzhaller) [reply seed/seed.json]: fixed as requested by @tmancey

**Review by iefremov — APPROVED** [review]

**Pull request merged**